foundry.toml

```
@@ -8,7 +8,7 @@ fs_permissions = [
     { access = "read", path = "./lib/morpho-blue/out/"}
 ]
 libs = ["lib"]
-optimization_runs = 999999 # Etherscan does not support verifying contracts with more optimization runs.
+optimizer_runs = 999999 # Etherscan does not support verifying contracts with more optimization runs.
```

```suggestion
optimizer_runs = 999999 # Etherscan does not support verifying contracts with more optimizer runs.
```
let's say that now, so less likely we do a mistake in the future